src/test/java/org/broadinstitute/consent/http/resources/TDRResourceTest.java

```diff
@@ -98,4 +115,51 @@ public void testGetDatasetByIdentifier404() {
         assertEquals(404, r.getStatus());
     }
 
+    @Test
+    public void testCreateDraftDataAccessRequest() {
+
+        String identifiers = "DUOS-00001, DUOS-00002";
```

Comment:
I'm not quite sure how this test is failing, but one thing I see is that you're splitting on the `,` but this string has an extra space after the comma, so you'd have two elements in your split array: `["DUOS-00001", " DUOS-00002"]` ... notice the extra space. That's not the source of the failure, but it is a potential bug. We should probably be trimming those values in the resource class.

Reply:
Got this test working and added a trim() when taking in the list of identifiers.